app/assets/javascripts/discourse/controllers/keyboard-shortcuts-help.js.es6

```diff
@@ -1,7 +1,123 @@
 import ModalFunctionality from "discourse/mixins/modal-functionality";
 
+const KEY = "keyboard_shortcuts_help";
+
+const SHIFT = I18n.t("shortcut_modifier_key.shift_lowercase");
+const ALT = I18n.t("shortcut_modifier_key.alt_lowercase");
+const CTRL = I18n.t("shortcut_modifier_key.ctrl_lowercase");
+const ENTER = I18n.t("shortcut_modifier_key.enter");
+const COMMA = I18n.t("number.format.delimiter") + " ";
+
+function trans(key, { sc = [], sc2 = [], deli = COMMA }) {
```

Comment:
what is the meaning of "trans" here ?

Reply:
translate. I'm going to rename it to `translate` since `trans` seems to be ambiguous. If you have a better name for it let me know.

Reply:
That's maybe too verbose given how much it's used but maybe something like `handleTextDirection` ?

Reply:
I like this suggestion! Will make it so, thanks!

Reply:
I deviated a little bit here and went with different names: `buildShortcut()`, `buildHTML()` and `wrapInSpan()`.

`buildHTML()` generates for example `<kbd>g</kbd>, <kbd>h</kbd>` and calls `wrapInSpan()` on that and returns it.
`buildShortcut()` uses `buildHTML()` output as the `%{shortcut}` variable for the the given translation key. Let me know if there is still ambiguous code.